### ISSUE TITLE
VisualSFM log error

### ISSUE THREAD

**Issue by kikislater** [pyp22/NuxSFM-2.0#20]
When I start Vsfm, in the log window, I have an error : 
```
ERROR: Unable to write to log file 
/usr/local/bin/log
```
Why the log should be in /usr/local/bin/log ?

As a temporary workaround, I modify the right of /usr/local/bin/log but I think it's not a good way ...

**Comment by pyp22** [comment]
Default VisualSFM log location is in its bin dir which is indeed not optimal. PKGBUILD solves that by a ln -sfv /var/log/VisualSFM "$pkgdir/usr/local/bin/log"

Does your user has perms to write in /var/log ?

**Comment by kikislater** [comment]
Yes and I have a log file in it. So I should ignore this warning, I have to test ...
